sympy/combinatorics/free_groups.py

```diff
@@ -146,7 +146,7 @@ def __new__(cls, symbols):
             obj = object.__new__(cls)
             obj._hash = _hash
             obj._rank = rank
-            # dtype method is used to create new instances of FreeGroupElement
+            # dtype method is used to create new instances of FgGroupElement
```

Comment:
I think this could be `FreeGroupElement` even if that derives from `FgGroupElemt`.